administration/src/components/verification/VerificationController.tsx

```diff
@@ -0,0 +1,124 @@
+import { Button, Card, H1, NonIdealState, Spinner } from '@blueprintjs/core'
```

Comment:
Although our file structure of the administration frontend is super messy (and should be refactored at some point), I think we should avoid putting the VerificationController into the `src/components` folder. Maybe, we should rather put it into the `src/application` folder as it's more related to the application stuff (and should use material ui instead of blueprintjs)? Alternatively into a new folder `src/application-verification` or so?

Reply:
I went with a separate folder, as `src/application` already encompasses the whole application form.